docker_compose/.env

```diff
@@ -0,0 +1,66 @@
+# Deployment configuration
+SC4SNMP_IMAGE=ghcr.io/splunk/splunk-connect-for-snmp/container
```

Comment:
most of variables here are set to their default values?

Reply:
yes, most of them are default values